src/app/template/index.ts

```diff
@@ -263,6 +263,19 @@ export class ChartTemplateBuilder {
         const plotSegment = elementClass.object as Specification.PlotSegment;
         table = plotSegment.table;
       }
+      if (Prototypes.isType(elementClass.object.classID, "links")) {
+        const linksTableName = this.dataset.tables[1].name;
```

Comment:
I think there should be a `elementClass.object.properties.linkTable.table` which will be the table name, so we don't need to assume that `tables[1]` is the links table.